tests/containers/testenv_prepare.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman
+# Summary: Running Podman nginx container under systemd
+# Create my_nginx.container file
+# Inform systemd about the new unit file and this creates a nginx.service
+# Start the created nginx service.
+# Verify the status of the nginx service.
+# Publish the port the container is running on.
+# Fetch a predefined website.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use containers::common;
+use containers::utils;
+use containers::container_images;
+
+sub run {
+    my ($self) = @_;
+    my $unit_name = 'my_nginx';
+
+    select_serial_terminal;
+    my $quadlet = script_output "rpm -ql podman | grep podman/quadlet";
+
+    assert_script_run('curl -sLf --create-dirs -vo /home/nginx/nginx.conf ' . data_url('containers/nginx/') . 'nginx.conf');
+    assert_script_run('curl -sLf --create-dirs -vo /home/nginx/index.html ' . data_url('containers/nginx/') . 'index.html');
+    assert_script_run('curl -sLf  -o /etc/containers/systemd/my_nginx.container ' . data_url('containers/nginx/') . 'my_nginx.container');
+
```

Comment:
LGTM..
But the file mane could be als less personale, being archived into the main repository, so it could even be like `nginx.containes` i.e.